client/src/components/event/filter-bar.js

```diff
@@ -50,8 +47,8 @@ class DropdownGroup extends React.Component {
   }
 }
```

Comment:
@yevchuk `window.location.reload()` reloads all page it isn't SPA approach. I've created task https://trello.com/c/uF3svZRL/96-91-fix-button-reset-in-events